packages/libdrm/libdrm.spec

```
@@ -0,0 +1,66 @@
+Name: %{_cross_os}libdrm
+Version: 2.4.128
+Release: 1%{?dist}
+Summary: Direct Rendering Manager runtime library
+License: MIT
+URL: https://dri.freedesktop.org
+Source0: https://dri.freedesktop.org/libdrm/libdrm-%{version}.tar.xz
+
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}glibc
+
+%description
+%{summary}.
+
+%package devel
+Summary: Files for development using the direct rendering manager library
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n libdrm-%{version} -p1
+
+%build
+CONFIGURE_OPTS=(
+ --auto-features=disabled
+ -Dcairo-tests=disabled
+ -Dman-pages=disabled
+ -Dvalgrind=disabled
+ -Dfreedreno=disabled
+ -Dvc4=disabled
+ -Detnaviv=disabled
+ -Dexynos=disabled
+ -Dtegra=disabled
+ -Domap=disabled
+ -Dintel=disabled
+ -Dradeon=enabled
+ -Damdgpu=enabled
+ -Dnouveau=disabled
+ -Dtests=false
+)
+
+%cross_meson "${CONFIGURE_OPTS[@]}"
+%cross_meson_build
+
+%install
+%cross_meson_install
+
+%files
+%{_cross_attribution_file}
+%{_cross_libdir}/libdrm.so.2
+%{_cross_libdir}/libdrm.so.2.128.0
```

We can probably * after the 2 otherwise these lines will be updated with every bump: 
```suggestion
%{_cross_libdir}/libdrm.so.2.*
```

---

Yes, make sense.